src/Tasks/Microsoft.NET.Build.Tasks/Resources/Strings.resx

```diff
@@ -121,7 +121,7 @@
     <value>At least one possible target framework must be specified.</value>
   </data>
   <data name="NoCompatibleTargetFramework" xml:space="preserve">
-    <value>Project '{0}' has no target framework compatible with '{1}'.</value>
+    <value>Project '{0}' has no target framework compatible with '{1}'. Compatible target frameworks are '{2}'.</value>
```

Comment:
I think I'd say something like:

Project '{0}' targets '{2}' but is being referenced from a project targeting '{1}' which is incompatible''